### DISCUSSION TITLE
Github status checks 'waiting' on Azure Pipelines

### DISCUSSION THREAD

**Discussion by planetf1** [community/community#23686] [New to GitHub]
<p>Hi,<br>
In <a href="https://github.com/odpi/egeria" rel="noopener nofollow ugc">https://github.com/odpi/egeria</a> we use Azure pipelines as our main build pipeline.</p>
<p>Since sometime yesterday we are seeing Status checks not completing - instead waiting for results to be reported. This has affected various different checks in pipelines, but our recently added CodeQL check, which uses github actions, is just fine.</p>
<p>There seems to be something very flaky in the communications between github and azure pipelines?</p>

**Comment by ernest-phillips** [comment]
<p>Hi <a class="mention" href="/planetf1">@planetf1</a> :wave:</p>
<p>Do you get an error when these checks fail?<br>
Can you list steps taken to create the pipeline? This information may help the community further troubleshoot the issue.</p>
<p>This is not a GitHub supported product, and I may suggest starting here in the Azure docs:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://docs.microsoft.com/en-us/azure/devops/pipelines/troubleshooting/troubleshooting?view=azure-devops" target="_blank" rel="noopener">docs.microsoft.com</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/5134583/181078762-2f6ad5fb-4ad3-497a-b313-0a4c03429fe7.png" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://docs.microsoft.com/en-us/azure/devops/pipelines/troubleshooting/troubleshooting?view=azure-devops" target="_blank" rel="noopener">Troubleshoot pipeline runs - Azure Pipelines</a></h3>

<p>Learn how to troubleshoot pipeline runs in Azure Pipelines and Team Foundation Server.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by planetf1** [comment, marked as answer]
<p>Hi,<br>
Indeed not github supported ,but it was difficult to know where the issue was as the problem was with the status reporting back to github.</p>
<p>In any case we decided it was a good time to try github actions instead, at least for PRs, so out of every challenge comes opportunity :slight_smile:</p>
<p>We now have our maven/gradle PR checks (+codeQL) running as actions. Nice…</p>
<p>Subsequently I understand it was an azure outage with pipelines that spanned across some of Thursday-Saturday</p>
